TFSIntegration/Script.ps1

```diff
@@ -0,0 +1,1609 @@
+﻿$leapworkHostname = Get-VstsInput -Name leapworkHostname -Require
+$leapworkPort = Get-VstsInput -Name leapworkPort -Require
+$leapworkAccessKey = Get-VstsInput -Name leapworkAccessKey -Require
+$leapworkTimeDelay = Get-VstsInput -Name leapworkTimeDelay
+$leapworkDoneStatusAs = Get-VstsInput -Name leapworkDoneStatusAs -Require
+$leapworkReport = Get-VstsInput -Name leapworkReport -Require
+$leapworkSchids = Get-VstsInput -Name leapworkSchids
+$leapworkSchedules = Get-VstsInput -Name leapworkSchedules -Require
+
+function Get-NewtonsoftJsonAssembly
+{   
+    return $PSScriptRoot, 'ps_modules', "Newtonsoft.Json", 'lib','net45','Newtonsoft.Json.dll' -Join '\' 
+}
+
+function Get-NetHttpAssembly
+{
+    return $PSScriptRoot, 'ps_modules', "System.Net.Http", 'lib','net46','System.Net.Http.dll' -Join '\'  
+}
+
+$newtonsoft = Get-NewtonsoftJsonAssembly
+$systemNetHttp = Get-NetHttpAssembly
+
+Write-Host "newtonsoft path: $newtonsoft"
+Write-Host "systemNetHttp path: $systemNetHttp"
+
+[Reflection.Assembly]::LoadFile($newtonsoft)
+[Reflection.Assembly]::LoadFile($systemNetHttp)
+
+$assemblies = @()
+$assemblies += $newtonsoft 
+$assemblies += $systemNetHttp
+$assemblies += "System.Runtime, Version=4.0.20.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a"
+$assemblies += "System.IO, Version=4.0.10.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a"
+$assemblies += "System.Xml.ReaderWriter, Version=4.0.10.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a"
+$assemblies += "System.Xml, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089"
+
+
+$sourceCode = @"
+using Newtonsoft.Json.Linq;
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.IO;
+using System.Net.Http;
+using System.Text;
+using System.Text.RegularExpressions;
+using System.Xml.Serialization;
+using System.Threading;
+using System.Threading.Tasks;
+
+
+namespace TFSIntegrationConsole
```

Comment:
Replace TFS to DevOpsAzure in code files

Reply:
Renamed to AzureDevOps